### ISSUE TITLE
Hangs when more than 3 CD's are listed

### ISSUE THREAD

**Issue by FatKiwi** [whiteisthenewblack/qobuz-get#2]
Example IDs: 0603497886531 or 0603497888207
Always hangs at CD4/Track1.

I see that you have to scroll in those pages for CD4 to fully show up. Is it related to that?

**Comment by whiteisthenewblack** [comment]
Yeah, you're right on the money there. Looks like I need to raise the limit on the number of tracks that can be requested. I'll commit a fix soon.

**Comment by FatKiwi** [comment]
Thanks, it will be helpful. Great work so far!

**Comment by whiteisthenewblack** [comment]
Seems like this was actually a quirk with D's `std.net.curl` library. Regardless, it should be fixed with https://github.com/whiteisthenewblack/qobuz-get/commit/fab4c19ad9891cbe5ad81b91001d885e502f5e2d.

Not creating a release atm, since fuwafuwa is down, but can you confirm that it works for you too? If you're using Windows i can send you a binary.

**Comment by FatKiwi** [comment]
Send over the binary. I'll test.

**Comment by whiteisthenewblack** [comment]
Here you go. Just replace the current qobuz-get.exe with this one.

[qobuz-get.zip](https://github.com/whiteisthenewblack/qobuz-get/files/1601585/qobuz-get.zip)

**Comment by FatKiwi** [comment]
Works brilliant, thanks. Please consider updating readme with compiling instructions. Thanks

**Comment by FatKiwi** [comment]
Cuts at CD4, track5:
core.exception.OutOfMemoryError@src\core\exception.d(696): Memory allocation failed
----------------

**Comment by whiteisthenewblack** [comment]
Sounds like your PC doesn't have enough free RAM...

**Comment by FatKiwi** [comment]
No, crashed again this time on track 3. I have 8GB free RAM.

**Comment by whiteisthenewblack** [comment]
That is strange. Testing it on Windows now.

**Comment by FatKiwi** [comment]
No problem, take your time.

**Comment by whiteisthenewblack** [comment]
As of https://github.com/whiteisthenewblack/qobuz-get/commit/169dcd9897440450a2dc2fe94fb0e736b4333c1e, I can no longer reproduce this on Linux. Issue _seems_ to be that on Windows, the pipe to ffmpeg sometimes break down for some reason.

There _might_ be a solution to this but the real solution would be to use libav rather than piping into ffmpeg. I'll try and address this as soon as I can.
